src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -82,21 +84,24 @@ const Body = BaseFormWithPolling.extend(Object.assign(
         return;
       }
 
-      if (this.isPollingRateLimitError(error)) {
+      if (this.isRateLimitError(error)) {
         // When polling encounter rate limit error, wait 60 sec for rate limit bucket to reset
         // before polling again & hide error message
-        setTimeout(() => {
-          model.trigger('clearFormError');
-        }, 0);
+        if (this.formSubmitted === false) {
```

Comment:
instead of using a client side boolean to keep status, we can use the current polling status we have
```
triggerAfterError(model, error) {
      BaseForm.prototype.triggerAfterError.apply(this, arguments);
     const isFormPolling = this.polling;
     this.stopPolling();

      if (error.responseJSON?.errorSummaryKeys?.includes('idx.session.expired')) {
        // Do NOT resume polling since session is invalid and polling is already stopped
        return;
      }

      if (this.isRateLimitError(error)) {
        // When polling encounter rate limit error, wait 60 sec for rate limit bucket to reset
        // before polling again & hide error message
        if (!isFormPolling) {
```

Reply:
sound good to me. Let me update and test it.

Reply:
Unfortunately, `saveForm` always call `this.stopPolling()` so we would still need additional variable in form state to know if it is user submitted.